frontend/src/pages/Player/RightPlayerPanel/components/NetworkResourcePanel/NetworkResourceLogs.tsx

```diff
@@ -67,6 +67,7 @@ export const NetworkResourceLogs: React.FC<{
 		logCursor: undefined,
 		startDate,
 		endDate,
+		disablePolling: true,
```

Comment:
Disable polling for network resource panel since this is locked into the time a session starts and ends